style.scss

```diff
@@ -29,15 +36,72 @@ body {
         .menu {
```

Comment:
Zamiast wielu mixinów zacząłbym od tego żebyś podzielił każdy element na osobny komponent. Czyli np `.menu {}` przenosisz do osobnego pliku do `sass/components/_menu.scss`. Pamiętaj że po wydzieleniu tego do osobnego pliku musisz to zaimportować w głównym pliku styli `style.scss` -> `import 'sass/components/menu'`